src/main/java/com/listywave/user/application/service/UserService.java

```diff
@@ -195,4 +215,43 @@ public void withdraw(Long userId) {
                 .map(Follow::getFollowerUser)
                 .forEach(User::decreaseFollowingCount);
     }
+
+    public void deleteLists(Long loginUserId, List<Long> listIds) {
+        List<ListEntity> lists = listRepository.findAllById(listIds);
+        if (lists.isEmpty()) {
+            throw new CustomException(RESOURCE_NOT_FOUND);
+        }
```

Comment:
만약 1개의 listId 만 DB에 값이 없다면, 존재하지 않는 1개에 대한 예외처리가 실패할 것 같습니다!

Reply:
음 만약 기존 1,2 에 대한 listId가 DB에 존재하는데 요청에 임의로 1,2,3을 보내거나 3을 보낸경우 아래 listRepository.findAllById 에서 해당 부분이 걸러져서 기존 DB에 없는 listId를 제외하고 가져와서 문제가 되보이진 않아 보입니다! 1,2,3 이 요청오면 1,2에 대해서만  List<ListEntity> lists 결과가 나오고 3이 요청이 오면 lists는 빈 컬렉션이 오게됩니다!

```suggestion
  public void deleteLists(Long loginUserId, List<Long> listIds) {
        List<ListEntity> lists = listRepository.findAllById(listIds);
        if (lists.isEmpty()) {
            throw new CustomException(RESOURCE_NOT_FOUND);
        }
```

Reply:
음.. 그럼 요청으로 들어온 listId에 해당하는 데이터가 아예 없는 경우만 검증한다고 이해하면 될까요?

Reply:
넵 이해한 바가 맞습니다!! ErrorCode를 따로 만들어 헷갈리지 않게 해야겠네요 ㅎㅎ